react-native-purchases-ui/src/index.tsx

```diff
@@ -339,7 +338,7 @@ export interface CustomerCenterCallbacks {
   /**
    * Called when a custom action is selected in the customer center.
    */
-  onCustomActionSelected?: ({actionId}: { actionId: string }) => void;
+  onCustomActionSelected?: ({actionId, purchaseIdentifier}: { actionId: string; purchaseIdentifier: string | null }) => void;
```

Comment:
I think this is breaking right? because `purchaseIdentifier` is not optional (even if it's nullable)